src/CLIHandler.ts

```diff
@@ -2,361 +2,362 @@
  * Copyright 2023 Kapeta Inc.
  * SPDX-License-Identifier: MIT
  */
-import * as Util from "util";
-import * as _ from "lodash";
-import blessed from "blessed";
-import OverviewEntry from "./cli/OverviewEntry";
-import { spawn } from "@kapeta/nodejs-process";
-import Symbols from "./cli/symbols";
+import * as Util from 'util';
+import * as _ from 'lodash';
+import blessed from 'blessed';
+import OverviewEntry from './cli/OverviewEntry';
+import { spawn } from 'cross-spawn';
```

Comment:
And I suppose this change wasn't strictly required, I was just diving in to try to eliminate sources of exitCode handling.